llmfoundry/callbacks/__init__.py

```diff
@@ -16,10 +17,6 @@
     ) from e
 
 __all__ = [
-    'FDiffMetrics',
-    'Generate',
-    'MonolithicCheckpointSaver',
-    'GlobalLRScaling',
-    'LayerFreezing',
-    'ScheduledGarbageCollector',
+    'FDiffMetrics', 'Generate', 'MonolithicCheckpointSaver', 'GlobalLRScaling',
+    'LayerFreezing', 'ScheduledGarbageCollector', 'ModelGauntlet'
```

Comment:
```suggestion
    'FDiffMetrics',
    'Generate',
    'MonolithicCheckpointSaver',
    'GlobalLRScaling',
    'LayerFreezing',
    'ScheduledGarbageCollector',
    'ModelGauntlet',
```

(adding the `,` at the end allows it to stay in one column; this is a nit/preference thing; doesn't actually matter if you don't want to include it)